### PR TITLE
[Plugin Creator] Remove manual (helpbrowser) from the plugin creator.

### PR DESCRIPTION
**Pull request by Marr11317**
Clicking on the help button now opens the online API.

As of recent discussion on telegram.

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
I guess we'd better remove plugins3.html and the whole shebang around it

File: mscore/plugin/pluginCreator.cpp [hunk 1]
```diff
@@ -558,7 +545,7 @@ void PluginCreator::textChanged()
 
 void PluginCreator::qmlWarnings(const QList<QQmlError>& el)
       {
-      foreach(const QQmlError& e, el)
+      for(const QQmlError& e: el)
```
Comment (Howard-C) [inline mscore/plugin/pluginCreator.cpp]: Please insert necessary spaces, like this: `for (const QQmlError& e : el)`.

**Comment by Marr11317** [comment]
hum, looks like I forgot to rerun Cmake...

**Comment by Marr11317** [comment]
not even: forgot to commit last changes...

**Comment by Marr11317** [comment]
> 
> 
> I guess we'd better remove plugins3.html and the whole shebang around it

done

File: mscore/plugin/pluginCreator.cpp [hunk 2]
```diff
@@ -578,17 +565,7 @@ void PluginCreator::msg(const QString& s)
...
-            manualDock = new QDockWidget(tr("Manual"), 0);
-            manualDock->setObjectName("Manual");
-
-            manualDock->setWidget(helpBrowser);
-            Qt::DockWidgetArea area = Qt::RightDockWidgetArea;
-            addDockWidget(area, manualDock);
-            helpBrowser->setContent(manualPath());
-            }
-      manualDock->setVisible(!manualDock->isVisible());
+      QDesktopServices::openUrl(QUrl("https://dmitrio95.github.io/plugins/html/index.html"));
```
Comment (dmitrio95) [inline mscore/plugin/pluginCreator.cpp]: Could you please replace the URL with this one?
https://musescore.github.io/MuseScore_PluginAPI_Docs/plugins/html/index.html
It looks like MuseScore-related domain is a better choice for permanent links like this :slightly_smiling_face:

Comment (Jojo-Schmitz) [reply mscore/plugin/pluginCreator.cpp]: Final cleanup before you leave the org I guess. Yes, should be done this way, in any case

Comment (Marr11317) [reply mscore/plugin/pluginCreator.cpp]: done

**Comment by Marr11317** [comment]
rebase needed

**Comment by Marr11317** [comment]
rebased

**Pull request merged**